pytest.ini

```diff
@@ -1,6 +1,4 @@
 [pytest]
```

Comment:
`[pytest]` maybe could be update into `pyproject.toml`.

Reply:
This sounds good. Would you prefer to sneak it in this PR or do it in a separate PR?

Reply:
I thought we better fix coverage problems first. So merge #1191 then merge this PR

Reply:
done at #1204